### DISCUSSION TITLE
mqqt light template - add color_mode

### DISCUSSION THREAD

**Discussion by otto2704** [home-assistant/architecture#1025] [Entity Models] [answered]
I am trying to implement configuration for fhem rgbww controllers [https://forum.fhem.de/index.php/topic,48918.msg619462.html#msg619462](url) (in german language).
Those controllers are mainly designed for usage with fhem, but also capable to be used with mqtt.
The best color model is hsv.
When using them with HA, mqtt light template is the best option, but this does not present a color selector in the UI and makes the thing pretty uncomfortable (see also here: [https://community.home-assistant.io/t/use-hs-color-for-mqtt-light-template/261014](url) ).
So it would be REALLY helpful if HA mqtt templates could also support color_mode.

**Comment by frenck** [comment]
If this something you are going to implement @otto2704? Or are you requesting a change?

**Comment by otto2704** [comment]
@frenck I am requesting a feature I think. Coming from fhem, I am still pretty new to HA

**Reply by frenck** [reply, marked as answer]
In that case, I'm going to close this discussion. This is not a place for requesting features, but for requesting/proposing architectural changes by our developers.

If you want to suggest a feature, you should try our [Community Forum: Feature Requests](https://community.home-assistant.io/c/feature-requests).

If you have additional questions, feel free to join our [Discord chat server](https://discord.gg/c5DvZ4e).

Thanks! 👍

**Reply by otto2704** [reply]
OK, thanks for this! As I said, I am new to HA...

**Reply by frenck** [reply]
Don't worry :)
